app/routes/user_routes.py

```
@@ -20,19 +20,19 @@
 # Create a new user
 # -----------------------------
 @router.post("/")
-def create_user(username: str, email: str, password: str, db: Session = Depends(get_db)):
+def create_user(payload: UserCreate, db: Session = Depends(get_db)):
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)